test/integration/ComputeFlow.test.ts

```diff
@@ -16,6 +16,7 @@ import { Account, DDO, Metadata } from '../../src/lib'
 import { Cluster, Container, Server } from '../../src/ocean/Compute'
 import { LoggerInstance } from '../../src/utils'
 import { ComputeInput } from '../../src/ocean/interfaces/ComputeInput'
+const fetch = require('node-fetch')
```

Comment:
no mixing of `require` & `import`. This should be:

```suggestion
import fetch from 'node-fetch'
```